client/app/components/Settings/Settings.js

```diff
@@ -131,7 +131,7 @@ class Settings extends Component {
           step={1}
           withBars
           value={this.state.settings.echo_range}
-          onChange={e => this.handleChange(e, 'echo_range')}
+          onAfterChange={e => this.handleChange(e, 'echo_range')}
```

Comment:
FYI @rahulbot -- waiting to fire the change events for the filters until after the sliders are released makes a big difference in the perceived responsiveness. The feed isn't constantly trying to update (e.g. filtering, API calls, etc.) as one drags the slider.